### PR TITLE
Promote console image for ODC+Serverless P2P testing

### PR DESCRIPTION
**Pull request by chetna14manku**
Promote console image to CI registry.
The image will be used in different config for running ODC+Serverless Integration tests.

Link to ticket: https://issues.redhat.com/browse/MSINTEROP-927

### CODE REVIEW

**Comment by chetna14manku** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images

**Comment by chetna14manku** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images

**Review by calebevans — APPROVED** [review]
/lgtm

**Comment by jerolimov** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images
/pj-rehearse auto-ack

**Comment by jerolimov** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images

**Comment by chetna14manku** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images

File: ci-operator/config/openshift/console/openshift-console-release-4.16__odc-serverless-p2p.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ocp_builder_rhel-9-base-nodejs-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-base-nodejs-openshift-4.16
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: console-release-4.16
+promotion:
+  to:
+  - name: ocp-console
```
Comment (jhadvig) [inline ci-operator/config/openshift/console/openshift-console-release-4.16__odc-serverless-p2p.yaml:31]: lets the image name more specific
```suggestion
  - name: serverless-4.16
```

or 

```suggestion
  - name: odc-serverless-4.16
```

**Review by jhadvig — COMMENTED** [review]
@chetna14manku PR looks good. Adding a comment about naming.
Also I see that no tests are specified in the config, is that on purpose? You only need to build an image and run the tests manually ?

**Comment by chetna14manku** [comment]
> @chetna14manku PR looks good. Adding a comment about naming. Also I see that no tests are specified in the config, is that on purpose? You only need to build an image and run the tests manually ?

@jhadvig thank you the review.
Our plan is to use this image in another config to run integration tests for both serverless+servicemesh and serverless+ODC.
This is WIP PR for this: https://github.com/openshift/release/pull/51135

File: ci-operator/config/openshift/console/openshift-console-release-4.16__odc-serverless-p2p.yaml (hunk 1 above)
Comment (chetna14manku) [reply ci-operator/config/openshift/console/openshift-console-release-4.16__odc-serverless-p2p.yaml]: Updated image name to `odc-serverless-4.16`

**Comment by chetna14manku** [comment]
/pj-rehearse pull-ci-openshift-console-release-4.16-odc-serverless-p2p-images

**Review by calebevans — APPROVED** [review]
/lgtm

**Comment by chetna14manku** [comment]
closing this PR, decided to use the already built image of console

**Pull request closed without merging**